cli/cargo-ohrs/src/cli/build.rs

```diff
@@ -0,0 +1,36 @@
+use bpaf::{construct, long, positional, Parser};
+
+use crate::util::Arch;
+
+pub fn cli_build() -> impl Parser<crate::Options> {
+  let dist = long("dist")
+    .argument::<String>("DIST")
+    .help("The path of the final build product is set to `dist` by default.")
+    .fallback(String::from("dist"));
+
+  let release = long("release")
+    .help("Build with release mode.")
+    .switch()
+    .fallback(false);
+
+  let arch = long("arch")
+      .short('a')
+      .help("The target build products support arm64/aarch, arm/arm32, and x86_64/x64 architectures, with all builds enabled by default.")
+      .argument::<Arch>("ARCH")
+      .many()
+      .optional();
```

Comment:
I'm not sure if `optional()` here and below does anything. `many()` collects values into a vector, when there's no arguments - you'll get an empty vector. `.optional()` would wrap it into `Some`.

Reply:
I am not sure that we can input none with `many` and get default value with `fallback`.

Reply:
`fallback` handles "value is missing" errors and substitutes a new value instead. `many` doesn't produce this kind of failure, but `some` does. If you want a fallback for no input try `.some("").fallback(xxxx)` or `.some("").optional()`.